lib/glimesh/graphql/schema/account_types.ex

```diff
@@ -32,6 +32,13 @@ defmodule Glimesh.Schema.AccountTypes do
       arg(:user_username, :string)
       resolve(&AccountResolver.all_followers/2)
     end
+
+    @desc "Count all follows or followers"
+    field :followers, list_of(:follower) do
+      arg(:streamer_username, :string)
+      arg(:user_username, :string)
+      resolve(&AccountResolver.count_followers/2)
+    end
```

Comment:
I think this approach would end up with us overwriting the existing followers() query. 

Instead it may be a good idea to add a countFollowers countFollowed to the existing User object? What are your thoughts?